tests/Db/Adapter/PdoAdapterTest.php

```diff
@@ -228,14 +229,14 @@ public function testShouldFetchConfig()
             ->will($this->returnValue(1));
 
         $adapter = new PdoAdapter($pdo);
-        $result  = $adapter->fetchByConfig($config, $pagination = 1);
+        $result = $adapter->fetchByConfig($config, $pagination = 1);
 
         $this->assertSame(1, $result);
     }
 
     public function testShouldFetchConfigUsingSpecifiedQuery()
     {
-        /** @var \Manager\Config\Node|\PHPUnit_Framework_MockObject_MockObject $pdo */
+        /* @var \Manager\Config\Node|\PHPUnit_Framework_MockObject_MockObject $pdo */
```

Comment:
lol, this is the second problem discovered in php-cs-fixer (after https://github.com/pennyphp/penny/issues/64) using php-cs-fixer